packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -11,6 +11,11 @@ import type {
 	CloudflareEnvironmentOptions,
 	CloudflareDevEnvironment,
 } from './cloudflare-environment';
+import {
+	getModuleFallbackCallback,
+	type ResolveIdFunction,
+} from './module-fallback';
+import { createIdResolver } from 'vite';
```

Comment:
Vite is already imported so this can be used as `vite.createIdResolver`.

```suggestion
```

Reply:
sorry this got auto imported and I missed it 👍